src/router/routes.js

```diff
@@ -13,19 +12,12 @@ export default [
         component: LandingPage,
     },
     {
-        path: '/table-with-tabs',
-        name: 'table_with_tabs',
-        component: TableWithTabs
-    },
-    {
-        path: '/table-with-tabs-2',
-        name: 'table_with_tabs-2',
-        component: TableWithTabs2
-    },
-    {
-        path: '/table-without-tabs',
-        name: 'table_without_tabs',
-        component: TableWithoutTab
```

Comment:
This will be added back right?

Reply:
No since it uses FTable, which actually uses divs rather than q-table

Let's move the fixing of components to another ticket and concentrate on upgrade for now

On this design, we can put these components to freedom-components repo and then let quasar boilerplate use freedom-components as an add on in the future

https://www.figma.com/proto/9DPvxK6RuHFfLqCKUvkpLY/Quasar-Boilerplate?page-id=326%3A281&node-id=470%3A3414&viewport=376%2C48%2C0.25&scaling=scale-down&starting-point-node-id=326%3A282

Reply:
This is a wrapper. q-table will be added on the slot of `content` inside f-table

@dvikas